### PR TITLE
LPS-170192 - [Raylife AP] Create and implement sort filter

### PR DESCRIPTION
**Pull request by pedroliveirav**
[LPS-170192](https://issues.liferay.com/browse/LPS-170192)

### CODE REVIEW

**Comment by pedroliveirav** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx [hunk 1]
```diff
@@ -101,6 +101,7 @@ const PoliciesTable = () => {
 		[]
 	);
 	const [checkedStateStatus, setCheckedStateStatus] = useState<boolean[]>([]);
+	const [sortPolicyByEndDate, setSortPolicyByEndDate] = useState('asc');
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx:104]: Hey, I believe that you forgot to type it.

**Review by joanagrego — APPROVED** [review]
Approved! You just need to type the "sortPolicyByEndDate" state and it's done. :white_check_mark:

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @pedroliveirav

**Comment by pedroliveirav** [comment]
ci:test:sf

**Comment by pedroliveirav** [comment]
ci:test:sf

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by pedroliveirav** [comment]
ci:forward

**Pull request closed without merging**